tests/Dfc.CourseDirectory.Web.Tests/Controllers/BulkUploadApprenticeshipsControllerTests.cs

```diff
@@ -167,16 +178,12 @@ public async Task IndexPost_Errors()
             VerifyNotUploaded();
         }
 
-        [Matcher]
-        private static Stream ItIsOriginalCsvStream(string originalCsv)
+        private static bool IsOriginalStream(string originalCsv, Stream stream)
```

Comment:
I think the name of this method needs reviewing; it's more the contents of the stream that's being compared than the stream itself. Perhaps `MatchesStreamContents` or something?

Reply:
I don't think this method should be seeking on the stream either.

Reply:
Pushed a new naming, what do you think?

Reply:
I like revised name

Reply:
Also pushed a commit to make the controller responsible for the memory stream position, rewinding it each time it passes it to other things. What do you think of that commit?